fix-hosts-file-on-boot/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+chia_root: "/home/ubuntu/.chia"
```

Comment:
Shouldn't need a chia_root for this script? This whole script could probably just live in like.. usr/local/bin or /opt/* or something?